headless/src/jsMain/kotlin/dev/fritz2/headless/components/combobox.kt

```diff
@@ -615,10 +624,9 @@ class Combobox<E : HTMLElement, T>(tag: Tag<E>, id: String?) : Tag<E> by tag, Op
             hook(openDropdown)
 
 
-            Window.keydowns.mapNotNull { event ->
-                if (internalState.current.opened && shortcutOf(event) == Keys.Tab) false
-                else null
-            } handledBy internalState.setOpened
+            Window.keydowns.filter { event ->
```

Comment:
I would suggest to use the filtering of `keydownsIf()`-function to apply the filtering *immediately* and just at the time the first value of the `Flow`  is collected.

You might check the whole component for similar keyboard event handlings? It is always a good idea to prefer the immediate setup of filtering and maybe behaviour controlling.

Reply:
Thanks for the heads-up! I implemented your feedback and identified one other occurrance that I was able to adjust. The two individual commits have been merged and the PR description has been adjusted.